src/otx/core/model/entity/classification.py

```diff
@@ -622,7 +654,7 @@ def _export_parameters(self) -> dict[str, Any]:
         export_params["via_onnx"] = False
         export_params["input_size"] = self.image_size
         export_params["onnx_export_configuration"] = None
-        export_params["output_names"] = ["logits", "saliency_map"] if self.explain_mode else None
+        export_params["output_names"] = ["logits", "feature_vector", "saliency_map"] if self.explain_mode else None
```

Comment:
What do you think about moving explain related parameters to src/otx/core/model/entity/classification.py?

ExplainableOTXClsModel ->
```
    @property
    def _export_parameters(self) -> dict[str, Any]:
        """Defines parameters required to export a particular model implementation."""
        export_params = super()._export_parameters
        export_params["output_names"] = ["logits", "feature_vector", "saliency_map"] if self.explain_mode else None
        return export_params
```

Reply:
I think it is possible, my logic was to put it near `export_params["onnx_export_configuration"]`. But I can move it up. Probably some other parameters also can be moved to the upper level class, @sovrasov?

Reply:
We have to add the parameters which make sense at the current level of hierarchy. In `ExplainableOTXClsModel` we already know what the outputs should be in explain mode, so we can move `export_params["output_names"]` there.

Other parameters here are implementation-specific (in out case that's mmpretrain), and I'd opt to leave them as is. Harim is working on torchvision wrapper, so we'd better keep parameters which could be not generalizable at lower levels.